zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/authorization/IdentitySetupInitializeTest.java

```diff
@@ -309,10 +309,12 @@ public void shouldCreateConfiguredMappings() {
     final var role = new RoleRecord().setName(UUID.randomUUID().toString());
     final var mapping1 =
         new MappingRecord()
+            .setId(UUID.randomUUID().toString())
             .setClaimName(UUID.randomUUID().toString())
             .setClaimValue(UUID.randomUUID().toString());
     final var mapping2 =
         new MappingRecord()
+            .setId(UUID.randomUUID().toString())
             .setClaimName(UUID.randomUUID().toString())
             .setClaimValue(UUID.randomUUID().toString());
```

Comment:
❓ Do you have an issue to allow users to configure an id when configuring the initial mappings? Currently we only allow them to configure a claim name and value. Since id is user provided as well, we must ensure users can configure these as well.

Reply:
https://github.com/camunda/camunda/issues/27820